recipes/pymonctl/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "PyMonCtl" %}
+{% set version = "0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Kalmat/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1ac3ed6f9bd593606213d7691d32f742892255238b57a9d4dd0a1b42c6b183d1
+
+build:
+  number: 0
```

Comment:
This can probably be noarch with the use of https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-packages-with-os-specific-dependencies